velox/exec/tests/utils/TpchQueryBuilder.cpp

```diff
@@ -104,7 +106,7 @@ void TpchQueryBuilder::readFileSchema(
   tableMetadata_[tableName].fileColumnNames = std::move(fileColumnNamesMap);
 }
 
-void TpchQueryBuilder::initialize(const std::string& dataPath) {
+void TpchQueryBuilder::initialize(const std::string& dataPath, const int32_t parquetPrefetch) {
```

Comment:
Please fix the formatting with `make format-fix`.

Reply:
make format-fix ran than you!